docs/l10nScreenshotsTests.md

```diff
@@ -2,14 +2,30 @@
 We are using [`screengrab`](https://docs.fastlane.tools/getting-started/android/screenshots/) which works with fastlane to automate the process of capturing screenshots. 
 All the l10n screenshots are generated through the ui tests. These particular tests run as part of the screenshots package (`app/src/androidTest/mozilla/fenix/ui/screenshots`)
 
-### Steps to Run / Build
+### Run tests locally from Android Studio
+Navigate to `app/src/androidTest/mozilla/fenix/ui/screenshots`, once in that directory, run the full test suite or a specific test by clicking on the `>` button.
+
+By running them manually you can check whether the test works or not but screenshots will not be saved. 
+
+### Run tests locally from command line
 1. Install the gem:
 `sudo gem install screengrab`
 
 2. From command line run: 
 `fastlane screengrab --test_instrumentation_runner "androidx.test.runner.AndroidJUnitRunner"` 
 
-Screenshots will be saved in the root directory: `fastlane/metadata/android` 
+The package configuration, apk paths as well as the locales are set in [Screengrab file](https://github.com/mozilla-mobile/fenix/blob/073fd8939067bc7a367d8db497bcf53fbd24cdd2/fastlane/Screengrabfile#L5).
+In case there is a change there the file has to be modified accordingly.
+Before launching that command, there has to be an emulator running.
+
+Once the build and tests finsish, screenshots will be saved in the root directory: `fastlane/metadata/android`
 If there is a failure and screenshots are not saved, it may be necessary to create these folders manually first.
 Currently screenshots are uploaded to [`gcloud`](https://storage.googleapis.com/l10n-fenix/2019-06-21/screenshots.html) once they are generated manually by running the previous command line.
-In the future this process will be automated too and documented here.
+
+## Run tests on CI
+Currently there is a cron job scheduled weekly to run these tests on Monday.
+It is set [here](https://github.com/mozilla-mobile/fenix/blob/5a8a7f549946fc8ad6ccf31f8c9c6bc2180aaed2/.cron.yml#L27). And the test results can be seen in [treeherder](https://treeherder.mozilla.org/jobs?repo=fenix&fromchange=67fb033f1bc8b772b64a1fda410632dd7e59450e&selectedTaskRun=NngGd-kXTtGGDpI9sMl2Bw.0), see previous link as an example.
+
```

Comment:
I suggest creating a separate ## TODO or ## TBD section here that explains that the following has not (and may not ever) be implemented and why.  It might also help to link out to Firefox-iOS project to illustrate that this would be standard across all of our projects, if the need were there to actually generate artifacts for Android as well.  That way it's clear why half of this work has been implemented but not finished